Common/Utility/DificultyUtils.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Terraria;
+using Terraria.ModLoader;
+
+namespace TheBereftSouls.Common.Utility
+{
+    public class DificultyUtils : ModSystem
+    {
+        public static bool Revengeance = false;
+        public static bool Death = false;
+        public static bool EternityMode = false;
+        public static bool MasochistMode = false;
+        public static bool LegendaryMode = false;
+
+        public override void PostUpdateWorld()
+        {
+            CheckDificulty();
+            base.PostUpdateWorld();
```

Comment:
Unnecessary call to an empty base method. Not the biggest deal in the earth but it hints towards the method doing other things (when it doesn't really do anything else) which leads to misunderstandings.

[tModLoader source code reference](https://github.com/tModLoader/tModLoader/blob/9a3717675250f48d32f1e4765fee41dcf067671a/patches/tModLoader/Terraria/ModLoader/ModSystem.cs#L211 "Terraria.ModLoader.ModSystem | Line 211")